resources/views/print/send.blade.php

```diff
@@ -0,0 +1,15 @@
+<div class="card">
+    <div class="card-header">@lang('print.transfer_money')</div>
+    <div class="card-body">
+        <form method="POST" action="{{ route('print.send') }}">
+            @csrf
+            <select class="selectpicker" data-live-search="true" id="receiver_id" name="receiver_id" title="@lang('print.user')">
+                @foreach ($users as $user)
+                <option value="{{ $user->id }}">{{ $user->name }}</option>
+                @endforeach
+            </select>
+            <input id="balance" name="balance" type="number">
```

Comment:
You could add a placeholder here (see #155), so the user knows what to write here. I think it looks even better on mobile.